config.xml

```diff
@@ -96,5 +96,5 @@
     <plugin name="cordova-plugin-network-information" spec="^2.0.1" />
     <plugin name="cordova-plugin-device-authentication" spec="https://github.com/dvsa/cordova-plugin-device-authentication" />
     <plugin name="cordova-plugin-appconfig" spec="1.1.4" />
-    <engine name="ios" spec="4.5.5" />
+    <engine name="ios" spec="^4.5.5" />
```

Comment:
Can this be kept as 4.5.5

Reply:
Not sure how that got changed. But will fix it at 4.5.5

Reply:
It changes all the time for me too.